### PR TITLE
normalize the docs nav menus

### PR DESCRIPTION
**Pull request by devoncarew**
- make the mav menu and the first page table of contents agree
- make the css for the first page and the rest of the pages the same

@sethladd

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 Flutter is a new way to build high-performance, cross-platform mobile apps.
-Flutter is optimized for today's, and tomorrow's, mobile devices. We are
-focused on low-latency input and high frame rates on Android and iOS.
+Flutter is optimized for today's, and tomorrow's, mobile devices. We are focused
+on low-latency input and high frame rates on Android and iOS.
 
-Next steps:
+## Next steps
 
+ - [FAQ](faq)
```
Comment (sethladd) [inline index.md]: Not sure that FAQ is a next step. :)

**Comment by sethladd** [comment]
Thanks! Since we're taking a look at this, not sure FAQ should be the first item in the top menu.

File: index.md (hunk 1 above)
Comment (devoncarew) [inline index.md]: It looked odd to me that this list was out of sync with the mav menu. I could not find a way to re-order the nav items - I had wanted to move FAQ to near the end of the menu.

I'll push this down the list, or move to the see also section.

**Comment by devoncarew** [comment]
Updated!

**Comment by devoncarew** [comment]
(updated FAQ position; will land)

**Pull request merged**